resources/langs/nheko_de.ts

```diff
@@ -6,89 +6,89 @@
+        <translation>Titel</translation>
     </message>
     <message>
         <location filename="../../src/dialogs/CreateRoom.cc" line="46"/>
         <source>Topic</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Thema/Beschreibung</translation>
     </message>
     <message>
         <location filename="../../src/dialogs/CreateRoom.cc" line="49"/>
         <source>Alias</source>
-        <translation type="unfinished"></translation>
+        <translation>Alias</translation>
     </message>
     <message>
         <location filename="../../src/dialogs/CreateRoom.cc" line="57"/>
         <source>Room Visibility</source>
-        <translation type="unfinished"></translation>
+        <translation>Raum Sichtbarkeit</translation>
     </message>
     <message>
         <location filename="../../src/dialogs/CreateRoom.cc" line="66"/>
         <source>Room Preset</source>
-        <translation type="unfinished"></translation>
+        <translation>Raum Vorlage</translation>
     </message>
     <message>
         <location filename="../../src/dialogs/CreateRoom.cc" line="76"/>
         <source>Direct Chat</source>
-        <translation type="unfinished"></translation>
+        <translation>Direkter Chat</translation>
     </message>
 </context>
 <context>
     <name>InviteUsers</name>
     <message>
         <location filename="../../src/dialogs/InviteUsers.cc" line="36"/>
         <source>CANCEL</source>
-        <translation type="unfinished"></translation>
+        <translation>ABBRECHEN</translation>
     </message>
     <message>
         <location filename="../../src/dialogs/InviteUsers.cc" line="47"/>
         <source>User ID to invite</source>
-        <translation type="unfinished"></translation>
+        <translation>Nutzer ID, die eingeladen werden soll</translation>
     </message>
 </context>
 <context>
     <name>JoinRoom</name>
     <message>
         <location filename="../../src/dialogs/JoinRoom.cc" line="30"/>
         <source>CANCEL</source>
-        <translation type="unfinished"></translation>
+        <translation>ABBRECHEN</translation>
     </message>
     <message>
         <location filename="../../src/dialogs/JoinRoom.cc" line="41"/>
         <source>Room ID or alias</source>
-        <translation type="unfinished"></translation>
+        <translation>Raum-ID oder -Alias</translation>
     </message>
 </context>
 <context>
     <name>LeaveRoom</name>
     <message>
         <location filename="../../src/dialogs/LeaveRoom.cc" line="29"/>
         <source>CANCEL</source>
-        <translation type="unfinished"></translation>
+        <translation>ABBRECHEN</translation>
     </message>
     <message>
         <location filename="../../src/dialogs/LeaveRoom.cc" line="39"/>
         <source>Are you sure you want to leave?</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```

Comment:
Translation missing.
I would suggest "Bist du sicher, dass du den Raum verlassen willst?"

Reply:
Whoops missed to type it in